src/main/java/de/retest/web/selenium/NoElementWithHighEnoughMatchFoundException.java

```diff
@@ -8,6 +9,9 @@
 
 	public NoElementWithHighEnoughMatchFoundException( final Element resultFromExpected ) {
 		super( "The element " + resultFromExpected
-				+ " from the Golden Master could not be matched with high enough confidence." );
+				+ " from the Golden Master could not be matched with high confidence > "
+				+ System.getProperty( Properties.ELEMENT_MATCH_THRESHOLD_PROPERTY )
+				+ ". To change required confidence, please set property "
+				+ Properties.ELEMENT_MATCH_THRESHOLD_PROPERTY );
```

Comment:
If we "expose" this property now, we should add it to the [docs](https://docs.retest.de/recheck/usage/configuration/#properties) and the `defaults.properties` (although we do not read this currently).

Reply:
Wouldn't want to put it into defaults.properties. But I documented it in https://github.com/retest/docs/pull/60...